linchpin_config.yml

```diff
@@ -2,11 +2,13 @@
 keystore_path: "keystore"
 outputfolder_path: "outputs"
 inventoryfolder_path: "inventory"
-
+# inventory playbooks are not necessary because inventory_type specific tasks are defined in inventory_gen role
 inventory_playbooks:
   - duffy_inventory.yml
-
 async: false
 async_timeout : 1000
 no_output : false
 schema: "ex_schemas/schema_v2.json"
+inventory_layouts_path: "inventory_layouts"
+inventory_outputs_path: "inventory_outputs"
+inventory_type: "aws"
```

Comment:
This was the reason I had the item based loop in the output_writer role, which you removed.

- include: "{{ item }}"
  with_items:
    - "{{ inventory_playbooks }}"
  when: state == "present" and inventory_layout is defined